### ISSUE TITLE
Screen Reader: Sign in: The logo image is not focused and announced

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77251]
## Action Performed:
Prerequisites:
- the user is not signed in

1. Using Android TalkBack or iOS VoiceOver, open the Expensify app
2. Swipe and try to focus the Expensify logo image

## Expected Result:
The logo image should be focused and announced, e.g.:
- 'Expensify'

## Actual Result:
The logo image of Expensify is not focused and announced.

## Area issue was found in:
Sign in

## Failed WCAG checkpoints
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User impact:
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested resolution:
Ensure the image is not hidden from the screen readers.
https://appt.org/en/docs/react-native/samples/accessibility-focusability

Add a descriptive accessibilityLabel, e.g.:
- 'Expensify'
https://appt.org/en/docs/react-native/samples/accessibility-label

## Environment(s)
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/d6e11c85-7515-4b1b-8210-e36ee7e9f61e)

https://github.com/user-attachments/assets/7be04d32-e006-40ab-a900-e6ff58d12778

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7004769
**Issue reported by:** NA
**Slack conversation:** NA
